### DISCUSSION TITLE
Programatically creating github projects

### DISCUSSION THREAD

**Discussion by rogarciaa** [community/community#21639] [Projects and Issues]
<p>Hello all,</p>
<p>I have been looking around github cli and others on how to use github actios to create github projects (boards).</p>
<p>Has anyone achieved this?</p>

**Comment by mickeygousset** [comment, marked as answer]
<p>I haven’t done it but there are REST API calls for it, so you could create something relatively easily:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/rest/reference/projects" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/rest/reference/projects" target="_blank" rel="noopener">Projects - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/rest/reference/projects" target="_blank" rel="noopener">//docs.github.com/en/rest/reference/projects</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
